ilastik/applets/dataSelection/opDataSelection.py

```diff
@@ -389,6 +390,47 @@ def get_provider_slot(self, parent: Operator) -> OutputSlot:
         return opReader.Output
 
 
+class DummyDatasetInfo(DatasetInfo):
+    """Special DatasetInfo for datasets that can't be found in headless mode
+    """
+
+    def __init__(self, *, project_file: h5py.File = None, **info_kwargs):
+        super().__init__(**info_kwargs)
+
+    @property
+    def legacy_location(self) -> str:
+        return "Outer Space"
+
+    @property
+    def effective_path(self) -> str:
+        return "Outer Space"
+
+    @property
+    def display_string(self) -> str:
+        return "Dummy Zero Array"
+
+    def to_json_data(self) -> Dict:
+        raise NotImplemented("Dummy Slots should not be serialized!")
+
+    def get_provider_slot(self, parent: Operator) -> OutputSlot:
+        opZero = OpZeroSource(shape=self.laneShape, dtype=self.laneDtype, axistags=self.axistags, parent=parent)
+        return opZero.Output
+
+    @classmethod
+    def from_h5_group(cls, data: h5py.Group, params: Dict = None):
+        laneShape = tuple(data["shape"])
+        default_tags = get_default_axisordering(laneShape)
```

Comment:
I wonder if this can bite us later on... This `DummyDatasetInfo` is supposed to be a stand-in for missing data, and that missing data might have different default axistags depending on the file type (e.g.: .png might be yxc, precomputed chunks might be cxy). This will probably never be a problem because the axistags are always saved into the .ilp files, and therefore the default_tags would never have to be used, but it's something to keep in mind.

Reply:
yes, good point. The nice thing, if we were to save them, would be more straight forward deserialization (without the `get_default_axisordering` workaround, which is, as you already stated meaningless since `axisorder` is always supplied).